polkadot/node/network/collator-protocol/src/validator_side_experimental/common.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+use polkadot_node_network_protocol::peer_set::CollationVersion;
+use polkadot_primitives::Id as ParaId;
+
+/// Maximum reputation score.
+pub const MAX_SCORE: u16 = 1000;
+
+/// Limit for the total number connected peers.
+pub const CONNECTED_PEERS_LIMIT: u16 = 300;
+
+/// Limit for the total number of connected peers for a paraid.
+/// Must be smaller than `CONNECTED_PEERS_LIMIT`.
+pub const CONNECTED_PEERS_PARA_LIMIT: u16 = 100;
+
+/// Maximum number of relay parents to process for reputation bumps on startup.
+pub const MAX_STARTUP_ANCESTRY_LOOKBACK: u32 = 20;
+
+/// Reputation bump for getting a valid candidate included.
+pub const VALID_INCLUDED_CANDIDATE_BUMP: u16 = 10;
```

Comment:
How does this work if para has 20 collators? They will all decay to 0 before they get a chance to build the next block.

Reply:
yeah, these are some pretty random values, I didn't spend much time for making some reasonable proposal yet